### ISSUE TITLE
docker pull error on latest image v2.2.1

### ISSUE THREAD

**Issue by frittenlab** [kubernetes-csi/node-driver-registrar#142]
I tried to pull the latest docker image from:

`docker pull k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.1`

This fails with:

``` docker pull k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.1
Error response from daemon: manifest for k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.1 not found: manifest unknown: Failed to fetch "v2.2.1" from request "/v2/sig-storage/csi-node-driver-registrar/manifests/v2.2.1".

**Comment by ialidzhikov** [comment]
@frittenlab , as of writing this comment, the latest available release is v2.2.0 (see https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v2.2.0) and there is no v2.2.1 release.

**Comment by frittenlab** [comment]
No worries I think there was a typo in the release notes. As I copied the docker pull command from there. Thanks
